### ISSUE TITLE
Question about the implementation of the generative loss

### ISSUE THREAD

**Issue by 17qingyu** [wangyu0627/GC-HGNN#1]
The paper mentions an autoencoder-based generative loss Lgen as part of the overall training objective. However, when reviewing the released code, we only observed two contrastive loss terms, namely the inter-view and intra-view contrastive losses, combined as:
`loss = self.alpha * inter_loss + self.beta * intra_loss`
We did not find an explicit implementation or integration of the autoencoder reconstruction loss in the current training pipeline. Could you please clarify whether Lgen is implicitly included in these loss terms, omitted in the released code, or planned to be added in a later update? Any clarification would be greatly appreciated.

**Comment by wangyu0627** [comment]
Thanks for your question. This is caused by a **version mismatch** between the paper and the currently released code.

In the present codebase, the training script only combines the inter-view and intra-view contrastive losses, while the autoencoder-based generative loss \(L_{\text{gen}}\) is not explicitly added to the final objective. However, the **autoencoder itself is fully implemented**, and the reconstruction-related computations are available in: code/module/autocoder.py

We will **update the repository in an upcoming version** to explicitly include the generative loss and fully align the implementation with the paper. Thanks for pointing this out.

**Comment by 17qingyu** [comment]
Thank you for the clarification and for your prompt response.
